picasso/src/main/java/com/squareup/picasso3/BitmapHunter.java

```diff
@@ -209,52 +212,70 @@ Bitmap hunt() throws IOException {
     }
 
     networkPolicy = retryCount == 0 ? NetworkPolicy.OFFLINE.index : networkPolicy;
-    RequestHandler.Result result = requestHandler.load(data, networkPolicy);
-    if (result != null) {
-      loadedFrom = result.getLoadedFrom();
-      exifOrientation = result.getExifOrientation();
-      bitmap = result.getBitmap();
 
-      // If there was no Bitmap then we need to decode it from the stream.
-      if (bitmap == null) {
-        Source source = result.getSource();
-        try {
-          bitmap = decodeStream(source, data);
-        } finally {
-          try {
-            //noinspection ConstantConditions If bitmap is null then source is guranteed non-null.
-            source.close();
-          } catch (IOException ignored) {
-          }
-        }
-      }
+    Bitmap bitmap = null;
+    CountDownLatch latch = new CountDownLatch(1);
+    try {
+      latch.await();
+    }
+    catch (InterruptedException ie) {
+      throw new IOException(ie);
```

Comment:
nit: use InterruptedIOException

Reply:
nit: move catch onto same line as the ending try bracket.

Reply:
done and done.